### PR TITLE
[PROGRAMMES-6776] Added content type for ATI Stats

### PR DESCRIPTION
**Pull request by benolot**
Ticket: https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6776

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Controller/Clips/ListController.php [hunk 1]
```diff
@@ -22,13 +22,15 @@ class ListController extends BaseController
     private const PARAMS_CLIPS_COUNT_KEY = 'clipsCount';
     private const PARAMS_PAGINATOR_PRESENTER_KEY = 'paginatorPresenter';
     private const PARAMS_SHOW_ALL_PID_KEY = 'showAllPid';
+    private const ATI_CONTENT_TYPE = 'list-clip';
```
Comment (Bobkidbob) [inline src/Controller/Clips/ListController.php]: ayyyyyyyyyyyyyyyyyyyyyyy

File: src/Controller/Profiles/IndexController.php [hunk 2]
```diff
@@ -15,6 +15,7 @@ class IndexController extends BaseController
     public function __invoke(CoreEntity $coreEntity, ProfileService $isiteService)
     {
         $this->setIstatsProgsPageType('profiles_list');
+        $this->setAtiContentType('list-datadriven');
```
Comment (Tasiobg) [inline src/Controller/Profiles/IndexController.php]:
```suggestion
        $this->setAtiContentType('list-profiles');
```
Was this supopse to be like that?

Comment (benolot) [reply src/Controller/Profiles/IndexController.php]: Yup, changed.

**Pull request merged**